codalab/rest/interpret.py

```diff
@@ -300,17 +300,17 @@ def fetch_interpreted_worksheet(uuid):
     # The bundle_infos for bundles that don't need updating are also None.
     if bundle_uuids:
         for i, block in enumerate(interpreted_blocks['blocks']):
```

Comment:
This looks good. Could we add a test within the `test/unit/rest` directory that ensures that the partial refresh endpoint never regresses again?

Reply:
Sure. The test has been added